### DISCUSSION TITLE
Offer different packages for different languages sucks

### DISCUSSION THREAD

**Discussion by gnokii** [inkstitch/inkstitch#1278] [General]
first of all doesnt go to my mind why there must be different packages at all, but I have to switch between english and my native language and documentation says, default would be english. So now I have Inkscapes locale at english but still have Ink/Stitch in my native language. So do I have to download really english and overwrite the files and when switching back to my native language do this again?
Localization is made so easy with .po files, what shall this?

**Comment by lexelby** [comment]
Yeah, it sucks.  We really wish we didn't have to do it this way!  The different language zips are _only_ for the Inkscape menus and the extension settings for a couple of the Ink/Stitch extensions.  Ink/Stitch itself (params, simulator, etc) uses your system language with po files and all that.

The reason we have to do it this way is that Inkscape doesn't make it possible for third-party extensions to do localization.  Extension INX files do support localization, but unfortunately, they only use a po file bundled with inkscape itself.  That means that only extensions that are actually bundled with inkscape can bundle their po files in.  It's a real shame.

**Reply by gnokii** [reply]
I not believe this, but honest one archive for all should be enough its just plain text so compressed nothing. This and the bundling of libs, makes it impossible to package it for any Linux distribution
